modules/charon-core/src/main/java/org/wso2/charon3/core/protocol/BulkRequestProcessor.java

```diff
@@ -161,6 +187,18 @@ public BulkResponseData processBulkRequests(BulkRequestData bulkRequestData) thr
                 }
             }
         }
+        // Handle v2 role operations.
+        for (BulkRequestContent bulkRequestContent : bulkRequestData.getRoleV2OperationRequests()) {
+            if (failOnError == 0) {
+                bulkResponseData.addRoleOperation(getBulkResponseContent(bulkRequestContent, userIdMappings,
+                        roleResourceV2Manager));
+            } else {
```

Comment:
We can have else-if

Reply:
Update the license header, when you send the commit.

Reply:
Addressed with https://github.com/wso2/charon/pull/407.